bindings/python/lexer.i

```diff
@@ -0,0 +1,22 @@
+%include "common.i"
+%include "fsm.i"
+
+%{
+#include <pire/re_lexer.h>
+using Pire::Lexer;
+%}
+
+class Lexer {
+public:
+    Lexer();
+
+    %extend {
+        Lexer(const char* begin, const char* end) {
+            Lexer* result = new Lexer(begin, end);
+            result->AddFeature(Pire::Features::AndNotSupport());
+            return result;
```

Comment:
- Fixed it with Pire::yauto_ptr.
- Yes, the generated code looks like this:
```c++
    try {
      result = (Lexer *)new_Lexer__SWIG_1((char const *)arg1,(char const *)arg2);
    }
    /* catching exceptions  */
  resultobj = SWIG_NewPointerObj(SWIG_as_voidptr(result), SWIGTYPE_p_Lexer, SWIG_POINTER_NEW |  0 );
```

The last line is exactly same as for `Lexer::Lexer()`, which is called directly from libpire.